<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ELECTOMENAGE</class>
 <widget class="QDialog" name="ELECTOMENAGE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back/images.png);</string>
  </property>
  <widget class="QTableView" name="tableView_electro">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Afficher">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TRI">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Etat : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_etat">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ajouter">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stat">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>statistique</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
